conf/dop/dop.yaml

```diff
@@ -7,9 +7,13 @@ http-server:
 
 grpc-server:
   addr: ":9529"
+i18n@cp:
+  common:
+    - conf/dop/i18n/cp/common.yaml
+  files:
+    - conf/dop/i18n/cp/scenarios/
 i18n:
```

Comment:
inject - conf/dop/i18n/resource-trans.yaml in cp

Reply:
I double checked the codes and it won't inflect translators injected through tag `translator:xxx` in DOP and CMP.